include/nuttx/lib/stdatomic.h

```diff
@@ -62,6 +62,109 @@
 #define ATOMIC_FLAG_INIT 0
 #define ATOMIC_VAR_INIT(value) (value)
 
+#ifdef CONFIG_LIBC_ARCH_NXATOMIC
+
+#define atomic_store_n(obj, val, type) \
+  (sizeof(*(obj)) == 1 ? __nx_atomic_store_1(obj, val, type) : \
```

Comment:
chang ALL `__nx_` to `nx_`

Reply:
@xiaoxiang781216 
Thank you for your advice.
I have done it.